### ISSUE TITLE
msg": "The 'file' lookup had an issue accessing the file 'configs/20.22.44.75/wireguard//.pki//private/phone'. file not found, use -vvvvv to see paths searched

### ISSUE THREAD

**Issue by Venkatesh7591** [trailofbits/algo#14687]
**Describe the bug**

I am getting an issue while running ./algo
After triggering all steps in this step i am facing issue can you please help me in this.

TASK [wireguard : Generate public keys] ************************************************************************************************************************************
task path: /home/tens/algo/roles/wireguard/tasks/keys.yml:80
fatal: [localhost]: FAILED! => {
    "msg": "The 'file' lookup had an issue accessing the file 'configs/20.22.44.75/wireguard//.pki//private/phone'. file not found, use -vvvvv to see paths searched"
}
looking for "config.cfg" at "/home/tens/algo/vars/config.cfg"
looking for "config.cfg" at "/home/tens/algo/config.cfg"
Read vars_file 'config.cfg'

TASK [include_tasks] *******************************************************************************************************************************************************
task path: /home/tens/algo/server.yml:123
looking for "config.cfg" at "/home/tens/algo/vars/config.cfg"
looking for "config.cfg" at "/home/tens/algo/config.cfg"
Read vars_file 'config.cfg'
included: /home/tens/algo/playbooks/rescue.yml for localhost
looking for "config.cfg" at "/home/tens/algo/vars/config.cfg"
looking for "config.cfg" at "/home/tens/algo/config.cfg"
Read vars_file 'config.cfg'
looking for "config.cfg" at "/home/tens/algo/vars/config.cfg"
looking for "config.cfg" at "/home/tens/algo/config.cfg"
Read vars_file 'config.cfg'

TASK [debug] ***************************************************************************************************************************************************************
task path: /home/tens/algo/playbooks/rescue.yml:2
ok: [localhost] => {
    "fail_hint": [
        "Sorry, but something went wrong!",
        "Please check the troubleshooting guide.",
        "https://trailofbits.github.io/algo/troubleshooting.html"
    ]
}
looking for "config.cfg" at "/home/tens/algo/vars/config.cfg"
looking for "config.cfg" at "/home/tens/algo/config.cfg"
Read vars_file 'config.cfg'

TASK [Fail the installation] ***********************************************************************************************************************************************
task path: /home/tens/algo/playbooks/rescue.yml:5
fatal: [localhost]: FAILED! => {
    "changed": false,
    "msg": "Failed as requested from task"
}

PLAY RECAP *****************************************************************************************************************************************************************
localhost                  : ok=71   changed=3    unreachable=0    failed=1    skipped=35   rescued=1    ignored=0

Thanks,
Venkatesh

**Comment by d1monch1k** [comment]
same error

**Comment by Venkatesh7591** [comment]
yeah issue is resolved

**Comment by Venkatesh7591** [comment]
Issue got resolved with myself.

**Comment by Venkatesh7591** [comment]
Issue got resolved myself.

**Comment by biosvs** [comment]
TLDR: Set `keys_clean_all: true` in config.cfg (it'll recreate keys for all user, beware!).

Seems like the problem is in lock files, they exist in cache (/etc/wireguard/ in my case), and it stops algo from recreating keys.
